### DISCUSSION TITLE
Promox, Clustering, UDMP, DNS - Happy places for automation to thrive

### DISCUSSION THREAD

**Discussion by jasonhollis** [autoSteve/acMqtt#48] [General]
I've been staring at the computer all day long.  I have forced myself away from the fun of looking at Proxmox until this point.  Been doing a bit of reading on it.  I noticed you mentioned the UDMP.  I have deployed lots and have the SE at mine.  I've been thinking about your DNS comments.  The local DNS/DHCP default setup from Ubiquiti sucks.  Used to build my own PFSense Opnsense boxes.  The default setups there with Unbound are much better but I found that the default ruleset in the UI firewall was probably a bit more practical and functional and certainly requires less maintenance.  I know you want to be cloud-free, but I have worked out that NextDNS does all and more in our environments than PI-Hole and takes about 5 minutes to set up.  It will use the same lists like HaGazi and a lot of other cool stuff.  It will also integrate right in with Tallscale and has an interesting free App for iOS and Android.

![CleanShot 2024-05-20 at 21 51 43@2x](https://github.com/autoSteve/acMqtt/assets/19743618/74259ebf-2a27-48ab-8e2a-56fc0fb94a6d)

I need to work out the correct model between the UI kit, DNS, Security, Tailscale HA, etc. Perhaps dropping Proxmox in a small cluster with HA on top and UI in front is the answer. I've been trying to keep it simple and only leverage the UDMP SE as much as possible, but the local DNS stuff is kind of killing me. It's also doing some wild and interesting things with IPV6 here and there.

On another complete aside, have you switched to Launtel yet?  If not, you need to check them out.  They are awesome.  They answer the phone in two rings, their people know what they are doing, they have a very cool routes out to the net, offer much faster speeds and have the unique ability for you to change your plan every day if you like.  I have moved about 20 people to them myself, and I have not been let down for a second yet.  On FTTP, I can go 1000/400 with them for $10/day

![CleanShot 2024-05-20 at 22 00 33@2x](https://github.com/autoSteve/acMqtt/assets/19743618/c65bc0a7-5743-4d73-96ab-18ac3b3cedfc)

When you call them, they will not ask you to unplug your UDMP and plug in the Telstra router. They know exactly what a UDMP is and will tell you how to optimise it if you don't know what you are doing, LOL.

**Comment by jasonhollis** [comment]
GF in Austin is moving house, and I was helping her work out her connectivity ... I got a bit sick with envy:

![IMG_4478](https://github.com/autoSteve/acMqtt/assets/19743618/19a00784-db18-44a1-8d8f-ca4860fb9765)

8Gb both ways for less than I'm paying for 1000/400.  This is how Google gets you.  As much as I don't like them, I think I would have to take this up and figure out how to put a tin foil hat on or something.

**Comment by autoSteve** [comment]
Launtel fascinates. I love their plan names. 🤭 Aussie have not let me down on 1000/50, so have not been sacked yet. HFC cable clips my wings...

Honestly, just 2Gbps symmetric would be heaven. Ah, heck, just give me bloody symmetric.

And good to know about NextDNS. Might be a great upsteam global forwarder choice for my local FreeIPA BGP-directed servers.

**Comment by jasonhollis** [comment]
I just noticed you’re paying slightly less on an annual contract etc. but the cool thing is that if I’m on holiday (and not streaming TV from my Plex server at home which will never happen) I could suspend it or turn it down and if I want more upload in the next five minutes I could be on 1000/400.  I know that’s meaningless to you on Hybrid at the moment but when you can make the swap you should consider them.  I don’t need the 1000/400 every day but there are points where I do flick over and back.
